<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPlayerClass</class>
 <widget class="QMainWindow" name="MPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QSlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>671</width>
      <height>211</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="beginTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="endTime">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="musicRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>音乐</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mvRadioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MV</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MPlayerClass</receiver>
   <slot>slot_play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MPlayerClass</receiver>
   <slot>slot_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MPlayerClass</receiver>
   <slot>slot_play_music(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MPlayerClass</receiver>
   <slot>slot_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MPlayerClass</receiver>
   <slot>slot_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_play()</slot>
  <slot>slot_search()</slot>
  <slot>slot_play_music(int,int)</slot>
  <slot>slot_stop()</slot>
 </slots>
</ui>
